_applications/equoid.adoc

```diff
@@ -0,0 +1,206 @@
+
+....
+oc new-app \
+    -l app=publisher \
+    -e OP_MODE={single|dual|linear} \
+    -e DATA_URL_PRIMARY=<YOUR_DATASET_URL> \
+    --image-stream=`oc project -q`/fuse-java \
+    https://github.com/radanalyticsio/equoid-data-publisher
+....
+
+
+=== Launch Data Handler
+
+We are now ready to start our data handler. Note that `INFINISPAN_HOST` and `INFINISPAN_PORT` need to be set to appropriate values, in the case of the template provided, these are `datagrid-hotrod` and `11222` respectively. In addition, `WINDOW_SECONDS` is the number of second for which you wish to track the top-k items, `SLIDE_SECONDS` represents the number of seconds to slide your data window each iteration, and `BATCH_SECONDS` is the size of samples you wish to acquire from the message stream at a time. Finally, `OP_MODE` needs to be set to the same value as set for the data publisher.
+
+....
+oc new-app --template=oshinko-scala-spark-build-dc \
+    -l app=handler \
+    -p SBT_ARGS=assembly \
+    -p APPLICATION_NAME=equoid-data-handler \
+    -p GIT_URI=https://github.com/radanalyticsio/equoid-data-handler \
+    -p APP_MAIN_CLASS=io.radanalytics.equoid.DataHandler \
+    -e INFINISPAN_HOST=<YOUR_HOSTNAME> \
+    -e INFINISPAN_PORT=<YOUR_PORT> \
+    -e WINDOW_SECONDS=<YOUR_WINDOW_SIZE> \
+    -e SLIDE_SECONDS=<YOUR_SLIDING_SIZE> \
+    -e BATCH_SECONDS=<YOUR_BATCH_SIZE> \
+    -e OP_MODE={single|dual|linear} \
+    -p SPARK_OPTIONS='--driver-java-options=-Dvertx.cacheDirBase=/tmp'
+....
+
+=== Launch Web UI and Set Up Keycloak
+
+Equoid's web UI is launched using a script developed by one of Equoid's contributors, Jiri Kremser. The following calls will utilize that script to set up the UI as well as a Keycloak instance for users and login. Finally, an edit role is given to the project's service account so additional data handler instances can be created/modified. 
+
+....
+BASE_URL="https://raw.githubusercontent.com/radanalyticsio/equoid-ui/master/ocp/"
+curl -sSL $BASE_URL/ocp-apply.sh | \
+    BASE_URL="$BASE_URL" \
+    KC_REALM_PATH="web-ui/keycloak/realm-config" \
+    bash -s stable
+oc policy add-role-to-user edit system:serviceaccount:`oc project -q`:default
+....
+
+[[usage]]
+== Usage
+
+=== Web Interface
+With all the microservices running, you can now view a visualization of the data stream's top-ks for the parameters you set. By browsing to the OpenShift webui for your cluster, logging in, and looking at your project's overview, you should see a pod/application named equoid-ui with an associated URI. By clicking on the latter, you will then be at the login screen for your equoid system. Log in using either of the default credential sets listed on the login page, then click on the graph tab. You should see a visualization of your stream's data which updates periodically. 
```

Comment:
Worth adding a screenshot of what we can expect to see in the visualisation?

Reply:
Agreed, added screen shots for the initial splash page and the graph page